packages/netcdf4/test_netcdf4.py

```diff
@@ -15,29 +15,29 @@ def test_netCDF4_tutorial(selenium):
 
     stdouts = [
         "NETCDF4",
-        "<class 'netCDF4._netCDF4.Dataset'>\nroot group (NETCDF4 data model, file format HDF5):\n    dimensions(sizes): \n    variables(dimensions): \n    groups: forecasts, analyses",
```

Comment:
This seems like a pretty bad test. Ideally we shouldn't care about the exact repr of all this junk. No action needed here, but in the future would be great to remove it or replace it with a better test.

Reply:
I agree, it took me a while to see what was going on here!